Probesys/bileto
# dev: Fix node bundler for uid != 1000

## Related issue(s)



N/A

## How to test manually



1. on your system, create a user with a uid !== 1000
2. follow the process to install Bileto in development mode https://github.com/Probesys/bileto/blob/main/docs/developers/setup.md
3. check that `make install` works

## Checklist
